web_decimal_numpad_dot/static/src/js/numpad_dot.js

```diff
@@ -1,18 +1,17 @@
 /* License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). */
 
-odoo.define('web_decimal_numpad_dot.FieldFloat', function (require) {
-"use strict";
+odoo.define("web_decimal_numpad_dot.FieldFloat", function (require) {
+    "use strict";
 
-    var core = require('web.core');
-    var form_widgets = require('web.form_widgets');
-    var translation = require('web.translation');
+    var form_widgets = require("web.form_widgets");
+    var translation = require("web.translation");
 
-    var FieldFloatDecimalDot = form_widgets.FieldFloat.extend({
-        events: {
-            'keypress': function (e) {
-                if (e.which === 46 || e.which === 44) {
-                    e.preventDefault();
-                    var input = this.$input || this.$('input');
+    var FieldFloatDecimalDot = form_widgets.FieldFloat.include({
```

Comment:
Remove `var FieldFloatDecimalDot = `, you're not using the variable. Not needed now.